prisma/migrations/20210316021800_add_sunshine_message_id_to_messages/migration.sql

```diff
@@ -0,0 +1,12 @@
+/*
+  Warnings:
+
+  - The migration will add a unique constraint covering the columns `[sunshineMessageId]` on the table `Message`. If there are existing duplicate values, the migration will fail.
+  - Added the required column `sunshineMessageId` to the `Message` table without a default value. This is not possible if the table is not empty.
```

Comment:
If the table were non-empty, this migration would fail right?

Reply:
yup, but since all the migrations are run in order there won't be a case where you run the first migration, then add data, and then run this one